NEWS.md

```diff
@@ -10,3 +10,5 @@
 * `@where` with a grouped data frame will now perform operations by group and filter
   rows in the parent `DataFrame`. The operation no longer filters groups. Returns a 
   fresh `DataFrame`.
+ * `@based_on` has been renamed to `@combine`
```

Comment:
is this only renaming?

Reply:
yes. just a find and replace with no new functionality.